src/main/java/com/ericsson/ei/frontend/utils/BackEndInstanceFileUtils.java

```diff
@@ -28,15 +32,25 @@
 
     private String eiInstancesPath;
 
+    @Autowired
+    BackEndInstancesUtils backendInstancesUtils;
+    
     @Value("${ei.backendInstancesFilePath:#{null}}")
     private String backendInstancesFilePath;
+    
+    @Value("${ei.backendInstancesListJsonContent:#{null}}")
+    private String backendInstancesListJsonContent;
 
     @PostConstruct
     public void init() throws IOException {
         LOG.info("Initiating BackEndInstanceFileUtils.");
 
+        JsonArray backendInstancesListJsonArray = (JsonArray) new JsonParser().parse(backendInstancesListJsonContent.toString());
```

Comment:
How does it handle invalid JSON?

Reply:
Added a try catch around the json parse step.

Reply:
Should it be handled more than just catching the issue and log? Should i crash or should it try and fix the structure?